backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe "spree/admin/shared/_navigation_footer", type: :view do
+  let(:user) { create(:admin_user) }
```

Comment:
`Spree.user_class.new` is quicker than factorygirl?  I guess it can be `build` instead of `create` if it doesn't need to persist, but I feel like it oughta be an admin user?